tests/bgp/test_bgp_peer_shutdown.py

```diff
@@ -151,6 +151,17 @@ def is_neighbor_session_down(duthost, neighbor):
             bgp_neighbors[neighbor.ip]["state"] == "idle")
 
 
+def get_bgp_down_timestamp(duthost, peer_ip):
+    # get the bgp session down timestamp from syslog in the format of seconds (with ms precision) since the Unix Epoch
+    cmd = (
```

Comment:
@cyw233 We may need to get the time from syslog only for time around the testing, there could be admin down/up event before or after this test.